example/index.js

```diff
@@ -1,7 +1,7 @@
 import * as rln from "@waku/rln";
 
 rln.create().then(async rlnInstance => {
-    let memKeys = rlnInstance.generateMembershipKey();
+    let credentials = rlnInstance.generateIdentityCredentials();
```

Comment:
```suggestion
    const credentials = rlnInstance.generateIdentityCredentials();
```
no?

Reply:
Also, didn't you/we said we could remove this example? cc @weboko

Reply:
That was on js-noise, although I think we can remove it here too?

Reply:
Before we were talking about `noise`'s example. 
We can remove that one but I'd keep it since it helps to test changes in one repo.